LoopKitUI/Views/ListMultipleSelection.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  ListMultipleSelection.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-02-27.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+public struct ListMultipleSelection<Item>: View where Item: Hashable, Item: CustomStringConvertible {
+    var items: [Item]
+    @Binding var selectedItems: Set<Item>
+    
+    public init(items: [Item], selectedItems: Binding<Set<Item>>) {
+        self.items = items
+        _selectedItems = selectedItems
+    }
+    
+    public var body: some View {
+        List(items, id:\.self) { item in
+            MultipleSelectionRow<Item>(item: item, selectedItems: self.$selectedItems)
+        }
+        .listStyle(GroupedListStyle())
+        .environment(\.horizontalSizeClass, .regular)
+    }
+}
+
+struct MultipleSelectionRow<Item>: View where Item: CustomStringConvertible, Item: Hashable {
+    var item: Item
+    
+    @Binding var selectedItems: Set<Item>
+    
+    var isSelected: Bool {
+        selectedItems.contains(item)
+    }
+    
+    var body: some View {
+        HStack {
+            Button(action: {
+                if self.isSelected {
+                    self.selectedItems.remove(self.item)
+                } else {
+                    self.selectedItems.insert(self.item)
+                }
+            }) {
+                Text(String(describing: item))
+                    .foregroundColor(.primary)
+            }
```

Comment:
It seems to me that we'd want the entire view to be tappable, rather than just the label.

Reply:
Yes, I agree. This is also the current behaviour I'm experiencing when testing.